src/components/Flyout/helpers/shouldDisplayAbove.ts

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MutableRefObject } from "react";
+
+type shouldDisplayAboveProps = {
+    triggerRef: MutableRefObject<HTMLDivElement | null>;
+    overlayHeight?: number;
+    margins: {
+        top: number;
+        bottom: number;
+    };
+    offset?: number;
+};
+
+export const shouldDisplayAbove = ({
```

Comment:
Is there a reason that you pass in an object instead of separate arguments? If not, I'd refactor the method